python-connectors/neo4j/connector.py

```diff
@@ -29,10 +31,7 @@ def generate_rows(self, dataset_schema=None, dataset_partitioning=None, partitio
                     query += " LIMIT {}".format(records_limit)
                 data = self.graph.run(query)
                 for row in data:
-                    node_properties = {
-                        "node_label": self.config["selected_node"],
-                        "neo4j_id": row["n"].identity
-                    }
+                    node_properties = {"node_label": self.config["selected_node"], "neo4j_id": row["n"].identity}
```

Comment:
Maybe replacing all the "n", "r", "name"... with a NODE, RELATIONSHIP... would make it clearer while keeping the actual queries short.